hikari/api/rest.py

```diff
@@ -6983,6 +7004,17 @@ async def create_context_menu_command(
 
             Defaults to `builtins.True`.
 
+            .. deprecated:: 2.0.0.dev109
+                Use `default_member_permissions` instead.
```

Comment:
Would clarifying that you can use `default_member_permissions` instead to get the same functionality better?

Reply:
But it isn't the same functionality, only similar, unless I understand this incorrectly.

Reply:
They're not even really similar tbh: v1 let bots control who can use their commands with `default_permission` deciding whether it could be used without explicit auth from the bot and v2 lets server moderators decide who can use commands and just lets bots suggest the initial perms; its a total inversion of v1 control wise and v2 just can't be used to achieve the behaviour of v1.

also random point but v2 broke default_permission (introducing a security hole for bots relying on it) by making it a suggestion rather than a strict rule so keeping it around is a bit iffy.

Reply:
Ok, will remove it then. Unfortunate it has to be this way :c

Reply:
Yeah its really annoying that discord did this with zero public notice but it is how it is